integration_tests/Dockerfile-wsgi-patch

```diff
@@ -0,0 +1,17 @@
+FROM python:3.9-slim-bullseye
+
+RUN apt update && apt install --assume-yes gcc
+
+COPY integration_tests/assets/bin/simple-wsgi-server.py /usr/local/bin
+COPY . /usr/local/src/xivo-lib-python
+
+RUN cd /usr/local/src/xivo-lib-python && \
+    python3 -m pip install -r requirements.txt && \
+    python3 -m pip install -r integration_tests/test-requirements.txt && \
+    python setup.py install
+
+# Allow 0 workes for cheroot server
+RUN sed -i 's/numthreads or 1/numthreads/' /usr/local/lib/python3.9/site-packages/cheroot/wsgi.py
```

Comment:
Because to be able to test the patch, we need all the workers to be dead. The easiest way to replicate that is to start the server without any, but it's impossible by default (the 'numthreads or 1' here). I'm using `sed` to remove that limitation and effectively start the server without any workers

Reply:
I think it's preferable to have a more realistic scenario. The web server is given a limited number of resources, and the goal is to see the behavior when it exceeds those constraints.

We don't want to prevent it to run normally, I don't think that reproduces the issue we're trying to fix.

Reply:
My understanding is that by constraining the file descriptors and/or disk space, and constraining the behavior of web workers(minimal number of workers that can easily be overloaded), we can reproduce the situation?

By making sure requests aren't processed quickly(a simple `sleep(5)` for example) and bombarding the server with requests, we should see the behavior we want?

Reply:
This bug shows up in 2 steps:
- first, the workers have to be hanging (which happen when the disk is full, they're waiting until they can write their logs)
- then, because the workers are stuck, each new incoming request open a file descriptor
- From there, the service has to exceed the maximum number of file descriptor allowed for the 5k lines of logs/sec to be written on the output